Integrations/FeedOffice365/FeedOffice365.yml

```diff
@@ -64,39 +63,33 @@ configuration:
   - F - Reliability cannot be judged
   required: true
   type: 15
-- display: ""
-  name: expiration
-  defaultvalue: ""
-  type: 17
+- name: expiration
   required: false
-- display: Bypass exclusion list
+  type: 17
+- additionalInfo: When selected, the exclusion list will be ignored for this feed.
```

Comment:
This field value was updated. Please look [here](https://demisto.slack.com/archives/CRT0G4Y82/p1579437446008100?thread_ts=1579435919.003300&cid=CRT0G4Y82) and update accordingly.